### DISCUSSION TITLE
Make Gltf clickable?

### DISCUSSION THREAD

**Discussion by erikpa1** [pmndrs/drei#978] [Q&A]
Hi guys I want to ask,

I have simple mesh layout editor and I want to activate models on click on them (I use useGLTF hook).

![image](https://user-images.githubusercontent.com/19526670/178281017-34184de3-e171-4a95-95bc-c8bd5964c31f.png)

**Comment by drcmda** [comment]
everything you have in your scene can receive clicks.

```jsx
<primitive object={scene} onClick={e => console.log("you clicked", e.object)} />
```

```jsx
<mesh onClick={e => console.log("you clicked", e.object)} geometry={...} material={...} />
```

```jsx
<group onClick={e => console.log("you clicked", e.object)} />
  <mesh ... />
  <mesh ... />
```

**Reply by erikpa1** [reply]
Oh thanks, `<group onClick={() => {}}>` is working very well, thank you.
